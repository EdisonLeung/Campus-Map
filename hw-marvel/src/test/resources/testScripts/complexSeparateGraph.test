# Test pathfinding on a complex graph where there are 2 groups of nodes that are
# disconnected

# Load Graph
LoadGraph separated separatedGraph.csv

# Pathfinding tests
FindPath separated a b

FindPath separated a c

FindPath separated d e

FindPath separated e f

FindPath separated b d

FindPath separated c f

FindPath separated e a

FindPath separated a a