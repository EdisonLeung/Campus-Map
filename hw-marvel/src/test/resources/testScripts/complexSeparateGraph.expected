# Test pathfinding on a complex graph where there are 2 groups of nodes that are
# disconnected

# Load Graph
loaded graph separated

# Pathfinding tests
path from a to b:
a to b via AB

path from a to c:
a to b via AB
b to c via BC

path from d to e:
d to e via DE

path from e to f:
e to f via EF

path from b to d:
no path found

path from c to f:
no path found

path from e to a:
no path found

path from a to a: